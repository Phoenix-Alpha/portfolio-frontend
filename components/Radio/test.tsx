import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { Radio } from '.'

describe('<Radio />', () => {
  const children = 'Test children'
  const value = 'Test value'

  it('should renders correct <Radio />', () => {
    const wrapper = shallow(<Radio value={value}>{children}</Radio>)

    expect(wrapper.find('.radio-button').prop('data-value')).toContain(value)

    expect(wrapper.contains(children)).toBeTruthy()

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
