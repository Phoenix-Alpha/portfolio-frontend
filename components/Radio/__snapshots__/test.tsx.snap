// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Radio /> should renders correct <Radio /> 1`] = `
<div
  className="radio-button"
  data-value="Test value"
>
  Test children
</div>
`;
